### PR TITLE
Add md exclude tags for slicing tutorial

### PR DESCRIPTION
**Pull request by brahmaneya**
**Test Plan**
tox. Also viewed in website locally

### CODE REVIEW

**Comment by henryre** [comment]
@brahmaneya definitely need to md-exclude the setup cell and corresponding text!

File: spam/03_spam_data_slicing_tutorial.py [hunk 1]
```diff
@@ -218,7 +220,7 @@ def textblob_polarity(x):
 # Most data points with high-polarity sentiments are strong opinions about the video — hence, they are usually relevant to the video, and the corresponding labels are $0$.
 # We might define a slice here for *product and marketing reasons*, it's important to make sure that we don't misclassify very positive comments from good users.
 
-# %%
+# %% {"tags": ["md-exclude-output"]}
```
Comment (henryre) [inline spam/03_spam_data_slicing_tutorial.py]: let's put slice_dataframe and head() in separate cells, then just exclude the slice_dataframe one

Comment (henryre) [reply spam/03_spam_data_slicing_tutorial.py]: looks like you did this above :+1:

**Comment by henryre** [comment]
* exclude output for trainer.fit ones
* add a proper printout with words when we score (0.928)
* there's an extra code cell at the bottom, rm

**Comment by brahmaneya** [comment]
> * exclude output for trainer.fit ones
> * add a proper printout with words when we score (0.928)
> * there's an extra code cell at the bottom, rm

I left the trainer fit output intentionally, because we want to show that it scores slices while training (we want to show that we can monitor slice performance while training). The second fit has even more slices being monitored.

Addressed all other comments

File: spam/03_spam_data_slicing_tutorial.py (hunk 1 above)
Comment (brahmaneya) [reply spam/03_spam_data_slicing_tutorial.py]: sorry missed this. doing now.

File: spam/03_spam_data_slicing_tutorial.py [hunk 2]
```diff
@@ -112,7 +114,7 @@ def short_link(x):
 
 sklearn_model = LogisticRegression(C=0.001, solver="liblinear")
 sklearn_model.fit(X=X_train, y=Y_train)
-sklearn_model.score(X_test, Y_test)
+print(f"Model Score: {sklearn_model.score(X_test, Y_test)}")
```
Comment (henryre) [inline spam/03_spam_data_slicing_tutorial.py]: `print(f"Test set accuracy: {100 * sklearn_model.score(X_test, Y_test):.1f}%")

**Comment by henryre** [comment]
@brahmaneya can you add the exact same `pd.set_option` block and comment from the lf and tf tutorials?

**Comment by brahmaneya** [comment]
> @brahmaneya can you add the exact same `pd.set_option` block and comment from the lf and tf tutorials?

Added
